### ISSUE TITLE
messenger:supervisor command

### ISSUE THREAD

**Issue by zorn-v** [symfony/symfony#35112]
Inspired by laravel horizon. 

Command that will run `messenger:consume` commands with parameters from config and watch that commands is running and start they if needed.
Maybe should use `symfony/lock` component for this.

**Comment by zorn-v** [comment]
Config may be like
```yaml
# config/packages/messenger.yaml
framework:
    messenger:
        supervisor:
            queue-1: ~
            queue-2:
                receivers: [receiver-1, receiver-2]
                memory-limit: 128M
                time-limit: 3600
                sleep: 1
                bus: mybus

```

**Comment by Nyholm** [comment]
Im not familiar with Laravel Horizon. 

Could you explain why this is needed? It looks like you are putting your supervisord configuration in the application.

**Comment by zorn-v** [comment]
> It looks like you are putting your supervisord configuration in the application.

Yes, more explaination in PR https://github.com/symfony/symfony/pull/35128

**Comment by Nyholm** [comment]
I don’t believe symfony should try to mimic all the excellence that supervisord brings. Is there a reason why you cannot put this config in supervisord? Ie, **why** do you need this feature?

**Comment by zorn-v** [comment]
As developer I can't edit supervisord config for example.

**Comment by Nyholm** [comment]
> As developer I can't edit (have no rights) supervisord config for example.

I believe that you have a suboptimal company policy. But even if this was the case, someone still needs to add config in supervisord for the new command you are introducing....

I dont believe this is the way Symfony should promote. 
However, since you are opening the issue, I do believe that this is a use case. I suggest you do like https://packagist.org/packages/m-adamski/symfony-schedule-bundle and create a bundle/library for this feature. 

Im 👎 for adding this feature to Symfony.

**Comment by zorn-v** [comment]
> But even if this was the case, someone still needs to add config in supervisord for the new command you are introducing....

And do it only ONCE per project

> and create a bundle/library for this feature.

But there is still missing feature in `messenger:consume` command - `--name` option added in PR

**Comment by fabpot** [comment]
I'm also :-1: on adding such a command. One need to use the numerous options out there. Let's close. Thanks for proposing.

**Comment by zorn-v** [comment]
Well, I create separate PR https://github.com/symfony/symfony/pull/35499 without this command. Is I need to create issue for it ?
